File: api-getting-data/getting_culture.py
require 'httparty'

/<author>/<Shakespeare>[..][/<lines>][,<title>][..][.<.json>]

import pyeuropeana.apis as apis
import pyeuropeana.utils as utils

response = apis.entity.suggest(
   text = 'Shakespeare',
   TYPE = 'agent',
)
print('tolkien', response)

File: api-getting-data/getting_culture.js
// api PoetryDB <author>/<Shakespeare>[..][/<lines>][,<title>][..][.<.json>]

async function suggestEntity() {
  try {
    const params = new URLSearchParams({
      text: 'Shakespeare',
      TYPE: 'agent'
    });

    const response = await fetch(`https://www.europeana.eu/api/entity/suggest?${params}`);
    const data = await response.json();
    
    console.log('Shakespeare', data);
    return data;
  } catch (error) {
    console.error('Error fetching data:', error);
  }
}

suggestEntity();
